lightning/src/chain/channelmonitor.rs

```diff
@@ -1630,7 +1630,7 @@ impl<Signer: Sign> ChannelMonitor<Signer> {
 	/// confirmations on the claim transaction.
 	///
 	/// Note that for `ChannelMonitors` which track a channel which went on-chain with versions of
-	/// LDK prior to 0.0.108, balances may not be fully captured if our counterparty broadcasted
+	/// LDK prior to 0.0.111, balances may not be fully captured if our counterparty broadcasted
 	/// a revoked state.
```

Comment:
FWIW, I thought there were other conditions where balances could be missing prior to 111 (dust htlc, preimage-not-yet-received)

Reply:
Oh, I meant the missing balances that were fixed in #1620 and #1691

Reply:
Ahh, gotcha. Sgtm